packages/shorebird_code_push_protocol/test/src/messages/create_patch_artifact/create_patch_artifact_request_test.dart

```diff
@@ -9,6 +9,7 @@ void main() {
         platform: ReleasePlatform.android,
         hash: '1234',
         size: 9876,
+        hashSignature: 'asd',
```

Comment:
It would be good to leave this test as it was before (all required fields only) and instead to add a new test where we add the optional `hashSignature` to reinforce the fact that this change is not breaking.